lib/assets/javascripts/turbograft/click.js

```diff
@@ -0,0 +1,50 @@
+/*
+ * decaffeinate suggestions:
+ * DS102: Remove unnecessary code created because of implicit returns
+ * DS207: Consider shorter variations of null checks
+ * Full docs: https://github.com/decaffeinate/decaffeinate/blob/main/docs/suggestions.md
+ */
+// The Click class handles clicked links, verifying if Turbolinks should
+// take control by inspecting both the event and the link. If it should,
+// the page change process is initiated. If not, control is passed back
+// to the browser for default functionality.
+window.Click = class Click {
+  static installHandlerLast(event) {
+    if (!event.defaultPrevented) {
+      document.removeEventListener('click', Click.handle, false);
+      return document.addEventListener('click', Click.handle, false);
```

Comment:
I think this can be omitted, especially since `document.addEventListener` [does not even return anything](https://developer.mozilla.org/en-US/docs/Web/API/EventTarget/addEventListener#return_value).
```suggestion
      document.addEventListener('click', Click.handle, false);
```